specification/mariadb/resource-manager/readme.typescript.md

```diff
@@ -7,6 +7,6 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/arm-mariadb"
-  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-mariadb"
+  output-folder: "$(typescript-sdks-folder)/sdk/arm-mariadb/arm-mariadb"
```

Comment:
Shouldn't that be mariadb according to the new convention?

Reply:
You're right that was a typo